pages/Mining.qml

```diff
@@ -37,6 +37,15 @@ Rectangle {
     color: "#F0EEEE"
     property var currentHashRate: 0
 
+    function isDaemonLocal() {
+        if (appWindow.currentDaemonAddress === "")
+            return false
+        var daemonHost = appWindow.currentDaemonAddress.split(":")[0]
+        if (daemonHost === "127.0.0.1" || daemonHost === "localhost")
+            return true
+        return false
+    }
```

Comment:
Depending on what this is used for, this will miss some local daemons. There is a function in src/common/util.h which does this (is_local_address).

Reply:
OK. This logic i also used on some other places. Added #939